w3f/polkadot_validator/galaxy.yml

```diff
@@ -0,0 +1,39 @@
+### REQUIRED
+
+# The namespace of the collection. This can be a company/brand/organization or product namespace under which all
+# content lives. May only contain alphanumeric lowercase characters and underscores. Namespaces cannot start with
+# underscores or numbers and cannot contain consecutive underscores
+namespace: w3f
+
+# The name of the collection. Has the same character restrictions as 'namespace'
+name: polkadot_validator
+
+# The version of the collection. Must be compatible with semantic versioning
+version: 0.0.2
+
+# The path to the Markdown (.md) readme file. This path is relative to the root of the collection
+readme: README.md
+
+# A list of the collection's content authors. Can be just the name or in the format 'Full Name <email> (url)
+# @nicks:irc/im.site#channel'
+authors:
+- Pantelis Ampatzoglou <pampatzoglou@gmail.com>
```

Comment:
Instead of individual users i think we should put the team as author, like here https://github.com/w3f/polkadot-secure-validator/blob/master/package.json#L6